operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -0,0 +1,263 @@
+          }
+      }
+    }
+    ```
+
+### Adding storage to a remote build node
+
+1. Add storage to the remote build node.
+
+    By default compute nodes have limited storage. While executing small image builds may be possible, you will not be able to build larger images without additional
+    storage being available to the IMS builder node. We can achieve this by mounting ceph storage into the IMS builder node.
+
+    Below are a set of commands that should provide the IMS builder node with the storage that it needs to execute larger image builds.
+
+    (`cn#`) Search for the static DNS list and servers and set the environment.
+
+    ```bash
+    egrep 'NETCONFIG_DNS_STATIC_SERVERS|NETCONFIG_DNS_STATIC_SEARCHLIST' /etc/sysconfig/network/config
+    NETCONFIG_DNS_STATIC_SEARCHLIST="search nmn mtl hmn"
+    NETCONFIG_DNS_STATIC_SERVERS="10.92.100.225"
+    ```
+
+    (`cn#`) Update the `netconfig` and install ceph-common.
+
+    ```bash
+    netconfig update -f
+    zypper install -y ceph-common
+    ```
+
+    (`ncn-mw#`) Copy over the ceph configuration and client admin keyring to the remote build node
+
+    ```bash
+    scp /etc/ceph/ceph.conf <compute node XNAME>:/etc/ceph/ceph.conf
+    scp /etc/ceph/ceph.client.admin.keyring <compute node XNAME>:/etc/ceph/
+    ```
+
+    (`cn#`) Configure ceph storage on the remote build node
+
+    ```bash
+    RBD=$( rbd map kube/buildcache --id kube --keyring /etc/ceph/ceph.client.kube.keyring )
+    echo $RBD
+    /dev/rbd1
+
+    mkdir -p /mnt/cache
+    mount /dev/rbd1 /mnt/cache
+
+    mkdir /mnt/cache/tmp
+    mount --bind /mnt/cache/tmp /tmp
+    mkdir -p /mnt/cache/var/lib/containers/storage/overlay/
+    mount --bind /mnt/cache/var/lib/containers/storage/overlay/ /var/lib/containers/storage/overlay/
+    ```
+
+### Adding and Removing Remote Build Nodes to IMS
+
+1. (`ncn-mw#`) Once your node has been created/configured with all of the above steps, the final step is to register that node with IMS so
+    that it knows that it can be used for image builds.
+
+    Adding a remote build node to IMS.
+
+    ```bash
+    cray ims remote-build-nodes create <compute node xname>
+    ```
+
+    Removing a remote build node from IMS.
+
+    ```bash
+    cray ims remote-build-nodes delete <compute node xname>
+    ```
+
+    List available remote build nodes.
+
+    ```bash
+    cray ims remote-build-nodes list
+    ```
+
+### Adding and removing a HSM Lock
+
+1. (`ncn-mw#`) Optionally lock the compute node to prevent unintended reboots of the compute node during operation.
+
+    Once the image is booted and operational you may also possibly look into the possibility of adding an HSM lock to that node. This will prevent unwanted or accidental reboots or power off.
+
+    ```bash
+    cray hsm locks lock create --component-ids <compute node XNAME>
+    ```
+
+    The lock can be removed at anytime via
+
+    ```bash
+    cray hsm locks unlock create --component-ids <compute node XNAME>
+    ```
```

Comment:
```suggestion
Optionally lock the compute node to prevent unintended reboots of the compute node during operation.

* (`ncn-mw#`) Create a node lock in HSM.

    ```bash
    cray hsm locks lock create --component-ids <compute node XNAME>
    ```

* (`ncn-mw#`) Remove a node lock in HSM.

    ```bash
    cray hsm locks unlock create --component-ids <compute node XNAME>
    ```

For more information, see [Manage HSM Locks](../hardware_state_manager/Manage_HMS_Locks.md).
```